recipes-bsp/u-boot/u-boot-iot2050.inc

```diff
@@ -68,6 +78,14 @@ do_prepare_build_append_secureboot() {
     cat ${WORKDIR}/secure-boot.cfg >> ${S}/configs/${U_BOOT_CONFIG}
 }
 
+do_prepare_build_append_rpmb-setup() {
+    if [ "${PRODUCT_GENERATION}" = "pg1" ]; then
+        bbwarn "PG1 devices are not supported in storing securely"
```

Comment:
This is too vague.

Reply:
Reworded to `PG1 devices do not supported RPMB based secure storage`